components/home/about/About.tsx

```diff
@@ -0,0 +1,113 @@
+import { Col, Container, Row } from "react-bootstrap";
+import { DatabaseNodes } from "./DatabaseNodes";
+import { ToolNodes } from "./ToolNodes";
+import styles from "./About.module.scss";
+import Highlight, { defaultProps } from "prism-react-renderer";
+import { WigglyLine } from "./WigglyLine";
+
+const code = `
+  // from https://nodejs.org/en/docs/guides/getting-started-guide/
+  const http = require("http");
+  const hostname = "127.0.0.1";
+  const port = 3000;
+`;
```

Comment:
We can use the real code here:
<img width="735" alt="Screen Shot 2021-12-09 at 4 51 02 PM" src="https://user-images.githubusercontent.com/303226/145498635-e92e19c6-99c0-465e-bce1-e29e9ac7f54f.png">